src/java/org/apache/cassandra/db/compaction/CompactionManager.java

```diff
@@ -148,8 +148,11 @@ public class CompactionManager implements CompactionManagerMBean
     public static final int NO_GC = Integer.MIN_VALUE;
     public static final int GC_ALL = Integer.MAX_VALUE;
 
-    //This is needed for Adaptive UCS to run in CNDB
-    public static final int PUBLISH_METRICS_INTERVAL = Integer.getInteger(Config.PROPERTY_PREFIX + "publish_metrics_interval_minutes", 0);
+    //This needs to be set on CNDB writers for Adaptive Compaction to work
+    public static final int PUBLISH_WRITER_METRICS_INTERVAL = Integer.getInteger(Config.PROPERTY_PREFIX + "publish_writer_metrics_interval_minutes", 0);
```

Comment:
I think we should prefix these with like `compaction.` or something similar, as they are only for these compaction metrics, not all metrics.
Also I would make the names make sense if CNDB did not exist.  For changes we make to the Cassandra code, if possible we want to be able to push them out to OSS C*.
Metrics often make sense for just a regular cassandra cluster, so this code is probably a good candidate for that?  and in that case we don't want to be talking about CNDB component names in it.